default/AI/ShipDesignAI.py

```diff
@@ -1924,6 +1910,77 @@ def _starting_guess(self, available_parts, num_slots):
             ret_val[-1] = num_slots - 1
         return ret_val
 
+
+def _add_shipdesign_fo(design_name, hull, parts_list, model="fighter", description="", icon="", name_desc_in_string_table=False, verbose=False):
```

Comment:
I would rather stick to the leading single leading underscore https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles - but I changed the function name